packages/models-library/src/models_library/projects_ui.py

```diff
@@ -24,10 +24,22 @@ class Config:
         extra = Extra.forbid
 
 
+class Annotation(BaseModel):
+    type: str = Field(..., description="Annotation type", examples=["rect", "text"])
```

Comment:
Suggestions:
- Add full example in the ``Config`` instead of partial this way we can automatically test the example with the object
- No need for descriptions if it is self explanatory by the context

TIP:  pydantic [undestands Literals as well](https://pydantic-docs.helpmanual.io/usage/types/#discriminated-unions-aka-tagged-unions) 
```
 class Annotation(BaseModel):
     type: Literal['rect', 'text'] = Field(..., description="Shape or text of the annotation")
```

Reply:
Done